httpcache_test.go

```diff
@@ -565,6 +565,8 @@ func (s *S) TestStaleIfErrorRequest(c *C) {
 	now := time.Now()
 	tmock := transportMock{
 		response: &http.Response{
+			StatusCode: http.StatusOK,
+			Status:     http.StatusText(http.StatusOK),
```

Comment:
Really minor style suggestion. In `net/http.Response`, `Status` comes first and it's followed by `StatusCode` ([source](https://godoc.org/net/http#Response)):

```Go
type Response struct {
    Status     string // e.g. "200 OK"
    StatusCode int    // e.g. 200
```

Maybe it'd be nicer to maintain that order? Feel free to disregard this if you don't think that'd be better.